sentry-samples/sentry-samples-spring-boot-jakarta-opentelemetry/src/main/java/io/sentry/samples/spring/boot/jakarta/PersonController.java

```diff
@@ -19,19 +20,27 @@
 @RequestMapping("/person/")
 public class PersonController {
   private final PersonService personService;
-  private final Tracer tracer;
+  private final OpenTelemetry openTelemetry;
   private static final Logger LOGGER = LoggerFactory.getLogger(PersonController.class);
 
-  public PersonController(PersonService personService, Tracer tracer) {
+  public PersonController(PersonService personService, OpenTelemetry openTelemetry) {
     this.personService = personService;
-    this.tracer = tracer;
+    this.openTelemetry = openTelemetry;
   }
 
   @GetMapping("{id}")
+  @WithSpan("personSpanThroughOtelAnnotation")
   Person person(@PathVariable Long id) {
-    Span span = tracer.spanBuilder("spanCreatedThroughOtelApi").startSpan();
+    ISpan annotationSpan = Sentry.getSpan();
+    System.out.println(annotationSpan);
```

Comment:
I believe these were just here for testing purposes, iirc

Reply:
updated in a follow up PR